kie-server-parent/kie-server-services/kie-server-services-kafka/src/test/java/org/kie/server/services/jbpm/kafka/KafkaServerExtensionTest.java

```diff
@@ -0,0 +1,37 @@
+package org.kie.server.services.jbpm.kafka;
+
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.kie.server.services.jbpm.kafka.KafkaServerUtils.KAFKA_EXTENSION_PREFIX;
+
+public class KafkaServerExtensionTest {
+
+    @Test
+    public void testProperties() {
+
+        KafkaServerExtension ext = new KafkaServerExtension();
+        System.setProperty(KAFKA_EXTENSION_PREFIX + ConsumerConfig.ALLOW_AUTO_CREATE_TOPICS_CONFIG, "false");
+        System.setProperty(KAFKA_EXTENSION_PREFIX + ProducerConfig.BATCH_SIZE_CONFIG, "1000");
+        System.setProperty(KAFKA_EXTENSION_PREFIX + "randomProperty", "JOJOJO");
+        try {
+            Map<String, Object> producerProperties = ext.getProducerProperties();
+            Map<String, Object> consumerProperties = ext.getConsumerProperties();
+            ext.initProperties();
+            assertEquals(1, producerProperties.size());
+            assertTrue(producerProperties.containsKey(ProducerConfig.BATCH_SIZE_CONFIG));
+            assertEquals(1, consumerProperties.size());
+            assertTrue(consumerProperties.containsKey(ConsumerConfig.ALLOW_AUTO_CREATE_TOPICS_CONFIG));
```

Comment:
You could also assert the value of these properties. 
"randomProperty" is not checked that it's not included in any of the properties.